admin/company/settings/sso-configuration.md

```diff
@@ -49,20 +54,20 @@ Follow the steps on this page to configure SSO for your company.
 
    ![Azure AD](../../../docker-hub/images/azure-create-connection.png){: width="500px" }
 
-4. From your IdP, copy and paste the following values into the Docker **Settings** fields:
+7. From your IdP, copy and paste the following values into the Docker **Settings** fields:
```

Comment:
I wonder if these needs an update since in Docker Hub SSO connection is listed under **Settings**, but it isn't in Docker Admin from what I could tell:
```suggestion
7. From your IdP, copy the following values to paste into the corresponding fields in the **Create Single Sign-on Connection** menu :
```